src/react/account/AccountPoliciesList.tsx

```diff
@@ -0,0 +1,331 @@
+      />
+    </SpacedBox>
+  );
+};
+
+const ActionButtons = ({
+  rowValues,
+  accountName,
+}: {
+  rowValues: InternalPolicy;
+  accountName: string;
+}) => {
+  const { arn, policyName, policyPath } = rowValues;
+  return (
+    <Box display="flex">
+      <AttachButton policyName={policyName} accountName={accountName} />
+      <EditButton
+        policyName={policyName}
+        policyPath={policyPath}
+        accountName={accountName}
+      />
+      <CopyARNButton text={arn} aria-label={`Copy ARN ${policyName}`} />
+      <DeletePolicyAction
+        policyName={policyName}
+        path={policyPath}
+        arn={arn}
+        accountName={accountName}
+      />
+    </Box>
+  );
+};
+
+const DeletePolicyAction = ({
+  policyName,
+  path,
+  arn,
+  accountName,
+}: {
+  policyName: string;
+  path: string;
+  arn: string;
+  accountName: string;
+}) => {
+  const dispatch = useDispatch();
+  const IAMClient = useIAMClient();
+  const [showModal, setShowModal] = useState(false);
+  const isInternalPolicy = path.includes('scality-internal');
+  const deletePolicyMutation = useMutation(
+    (arn: string) => {
+      return notFalsyTypeGuard(IAMClient).deletePolicy(arn);
+    },
+    {
+      onSuccess: () =>
+        queryClient.invalidateQueries(
+          getListPoliciesQuery(accountName, notFalsyTypeGuard(IAMClient))
+            .queryKey,
+        ),
+      onError: (error) => {
+        try {
+          dispatch(handleClientError(error));
+        } catch (err) {
+          dispatch(handleApiError(err as ApiError, 'byModal'));
+        }
+      },
+    },
+  );
+
+  return (
+    <>
+      <DeleteConfirmation
+        show={showModal}
+        cancel={() => setShowModal(false)}
+        approve={() => {
+          deletePolicyMutation.mutate(arn);
+        }}
+        titleText={`Permanently remove the following policy ${policyName} ?`}
+      />
+      <Box ml="0.6rem">
+        <Button
+          style={{ height: spacing.sp24 }}
+          disabled={isInternalPolicy}
+          icon={<i className="fas fa-trash" />}
+          label=""
+          onClick={() => {
+            setShowModal(true);
+          }}
+          variant="danger"
+          tooltip={{
+            placement: 'top',
+            overlay: isInternalPolicy
+              ? 'You cannot delete a predefined Scality Policy'
+              : 'Delete',
+          }}
+          aria-label={`Delete ${policyName}`}
+        />
+      </Box>
+    </>
+  );
+};
+
+const AccessPolicyNameCell = ({ rowValues }: { rowValues: InternalPolicy }) => {
+  const { policyPath, policyName } = rowValues;
+  const isInternalPolicy = policyPath.includes('scality-internal');
+  return (
+    <>
+      {isInternalPolicy && (
+        <Tooltip
+          overlay={'This is a predefined Scality Policy'}
+          overlayStyle={{
+            width: '13rem',
+          }}
+        >
+          {policyName} <Icon className="fas fa-question-circle fa-xs"></Icon>
+        </Tooltip>
```

Comment:
nit - can wait for later - can be ignored for now.

I wonder wether we already have(in zenko-ui) or should have such Tooltip+questionCircle icon component in core-ui. Something like `<HelperText overlay={string}>{icon => <>Toto {icon}</> || 'toto'}</HelperText>`
if the children props is a function, we pass it the icon so that it can be posisitioned whereever it is needed. If it is a string then the icon is automatically added next to it. If it is a component then a flex box wrap it automatically and the icon is centered left to that component.